src/Umbraco.Infrastructure/Examine/NoopBackOfficeExamineSearcher.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using Examine;
+using Umbraco.Examine;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Infrastructure.Examine
+{
+    public class NoopBackOfficeExamineSearcher : IBackOfficeExamineSearcher
+    {
+        public IEnumerable<ISearchResult> Search(string query, UmbracoEntityTypes entityType, int pageSize, long pageIndex, out long totalFound,
+            string searchFrom = null, bool ignoreUserStartNodes = false)
+        {
+            totalFound = 0;
+            return new ISearchResult[0];
```

Comment:
Should return a static (empty) instance of this instead of creating a new instance each time